### PR TITLE
Fix app.get always returning nil pr-fix

### PR DESCRIPTION
**Pull request by amlestin**
for https://github.com/Hammerspoon/hammerspoon/issues/3206

pr-fix

### CODE REVIEW

**Comment by andmis** [comment]
I'd love to see this get merged at some point!

**Comment by amlestin** [comment]
> I'd love to see this get merged at some point!

Do you know how I can add a pr-fix label to fix the workflow?

**Comment by andmis** [comment]
> > I'd love to see this get merged at some point!
> 
> Do you know how I can add a pr-fix label to fix the workflow?

Nope, unfortunately I don't. I'm not a maintainer or anything, just a bystander / cheerleader!

**Comment by amlestin** [comment]
> > > I'd love to see this get merged at some point!
> > 
> > 
> > Do you know how I can add a pr-fix label to fix the workflow?
> 
> Nope, unfortunately I don't. I'm not a maintainer or anything, just a bystander / cheerleader!

Thanks anyway :) I think only the repo owner can edit it.

**Comment by latenitefilms** [comment]
I believe only @asmagill and @cmsj have the power to assign labels.

**Comment by cmsj** [comment]
Added the label and kicked the CI off, not that I would particularly expect the results to be relevant here (and don't worry if there are unrelated failures - some of our tests are just unreliable, unfortunately!)

File: extensions/json/test_json.lua [hunk 1]
```diff
@@ -18,7 +18,10 @@ function testEncodeDecode()
 end
 
 function testEncodeDecodeFailures()
-  hs.json.encode(notATable)
+  local status, err = pcall(hs.json.encode(notATable))
```
[the remark this answers is not in the document]
Comment (amlestin) [reply extensions/json/test_json.lua:21]: Need variable err since a value is returned by pcall. Feel free to change if Lua has a convention like Go's _

**Comment by amlestin** [comment]
@cmsj that failing test was blocking the pipeline

**Comment by cmsj** [comment]
Thanks, and sorry this has taken so long!

**Pull request merged**